odata-protocol/11.4 Data Modification.md

```diff
@@ -69,10 +69,25 @@ specifying `If-Match` with a value of `*`. Services MAY reject such
 requests.
 
 For requests including an [`OData-Version`](#HeaderODataVersion) header
-value of `4.01`, any ETag values specified in the request body of an
-[update request](#UpdateanEntity) MUST be `*` or match the current value
+value of `4.01`, any ETag values specified in the request body of a
+[request for modifying an entity](#UpdateanEntity) MUST be `*` or match the current value
 for the record being updated.
 
+::: example
+Example ##ex_antietag: `Category` is a single-valued nullable non-containment navigation property
```

Comment:
Note that, (in the given case), the etag used in an if-match or if-not-match header applies to the category currently associated with the product. if you want to make an update conditional on the product, you would instead issue the update against the product, with the updates to the category in the payload.